examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
<p>Could you explain a bit more? How good/bad (in terms of security) is to use a semi-honest or malicious security? Which are the possible Protcols? I see <code>Falcon</code>  and <code>Function Secret Sharing</code>  ? (maybe I am wrong). It would be nice to include as well the link to the papers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Sure will do that :)